mcp_server/server.py

```
@@ -0,0 +1,1624 @@
+    logger.info(f"[MCP Server] Closing {len(_mcp_clients)} remote connection(s)...")
+    for server_url, client in _mcp_clients.items():
+        await client.close()
+    _mcp_clients.clear()
+    logger.info("[MCP Server] ✅ All connections closed")
+    logger.info("=" * 60)
+
+
+# 初始化 FastAPI 应用，使用 lifespan 事件处理器（必须在路由定义之前）
+app = FastAPI(title="Simple MCP Server", version="1.0.0", lifespan=lifespan)
+
+# 挂载静态文件（UI文件）
```

Ruff (0.14.8)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)